projects/applicationinsights-angularplugin-js/src/lib/applicationinsights-angularplugin-js.component.ts

```diff
@@ -76,7 +76,10 @@ export class AngularPlugin extends BaseTelemetryPlugin {
                     isPageInitialLoad = false;
                     return;
                 }
-                const pageViewTelemetry: IPageViewTelemetry = { uri: extConfig.router.url };
+                const pageViewTelemetry: IPageViewTelemetry = {
+                  uri: extConfig.router.url,
+                  properties: { duration: 0 } // SPA route change loading durations are undefined, so send 0
```

Comment:
Note: DO NOT set duration to 0 on initial page view (line 68)

Reply:
As part of the SDK initialization I believe it calls trackPageView() automatically...